src/layers/GridLines.tsx

```diff
@@ -212,3 +225,37 @@ const WeekStripes = ({ monthStart, timeScale }: WeekStripesProps) => {
 
   return <g>{lines}</g>
 }
+
+/* ·················································································································· */
+/*  Day
+/* ·················································································································· */
+
+interface DayViewProps extends Omit<Props, 'smallerZoomScale'> {}
+
+const DayView = ({ height, domain, timeScale }: DayViewProps) => {
+  const xAxisTheme: XAxisTheme = useTimelineTheme().xAxis
+  const textStyle = useYearViewTextStyle()
```

Comment:
Declare a `useDayViewTextStyle` which should also encapsulate `fontSize` (line 248)